### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by dschreij** [studiointeract/accounts-semantic#5]
I am just wondering: Seeing that the last commits to this repo have been made 9 months ago, is this project still maintained? Or has it moved to a different repo/user? Semantic is still very popular so it seems a bit of a pity to let this module slip away. Time pending, I can maybe put in some effort bringing it up to date, but I'm currently only using material-ui for a project and am not working on any semantic-ui projects at the moment. (might do so soon though).

TL;DR: I am wondering where this project is at and may be able to help.

**Comment by kevinrue** [comment]
Any update on this one? Again, the std:accounts-*  modules are really appealing, but they seem to suffer from recent changes affecting appearance and functionalities.

**Comment by derwaldgeist** [comment]
Was wondering the same. But then saw there's a successor here:
https://github.com/studiointeract/accounts-ui/
Would be great if this was mentioned in the docs.

**Comment by dschreij** [comment]
It's not a successor, its the package on which this one is based and supplies the styling for (std:accounts-ui is unstyled). As it says in the docs of accounts-ui:

> This package does not by standard come with any styling, you can easily extend and make your own, here are a couple versions we've made for the typical use case:
>
>  - Semantic UI std:accounts-semantic

**Comment by derwaldgeist** [comment]
Ok, thanks, understood :-)

**Comment by dschreij** [comment]
Sorry, that wasn't meant to come over that harsh. I think all std styling packages are due for an update, but often the fixes are easy to do yourself, if you really want to use them.

**Comment by derwaldgeist** [comment]
No prob, I didn't understand it harsh :-)
